peers/validators/canonical_validator_client.go

```diff
@@ -36,18 +36,10 @@ func NewCanonicalValidatorClient(logger logging.Logger, apiConfig *config.APICon
 	}
 }
 
-func (v *CanonicalValidatorClient) GetCurrentCanonicalValidatorSet(
+func (v *CanonicalValidatorClient) GetCanonicalValidatorSet(
```

Comment:
This looks like a pure passthrough to `avalancheWarp.GetCanonicalValidatorSet` now. Thoughts on removing it enitrely?